app/api/games/[id]/route.tsx

```diff
@@ -1,6 +1,6 @@
 import prisma from "@/prisma/prisma";
 
-export async function PATCH(request: Request) {
+export async function GET(request: Request) {
```

Comment:
Great that you've added separate GET for single game, but I guess you haven't mean to remove the PATCH? :D